courses/free-courses-en.md

```diff
@@ -771,6 +771,7 @@
 
 ### Graph Theory
 
+* [Graph Theory](https://www.youtube.com/playlist?list=PLEAYkSg4uSQ2fXcfrTGZdPuTmv98bnFY5) - Graph Theory Lectures by Prof. Soumen Maity teaching at IISER
```

Comment:
```suggestion
* [Graph Theory - Soumen Maity \| IISER - NPTEL](https://www.youtube.com/playlist?list=PLEAYkSg4uSQ2fXcfrTGZdPuTmv98bnFY5) - Graph Theory
```

Reply:
Don't invent titles! Use the title given by the resource.

Reply:
Done